docs/tutorials/creating-your-first-substrate-chain/setup.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Setup Your Computer
+---
+
+Normally we would teach you more about the Substrate blockchain development
+framework and the Proof of Existence blockchain you will be building. However,
+setting up your computer for Substrate development can take a while. 
+
+To optimize your time, we will have you start the setup process. In the next
+section, while things are compiling, we will then tell you more about Substrate
+and what we are building.
+
+## Prerequisites
+
+To develop on Substrate, your computer needs some prerequisites to establish a
+working development environment.
+
+> **Note:** Setting up your computer is probably the hardest part of this
+> tutorial, so don't let this discourage you.
+
+### Substrate Development
+
+If you are using a Unix based machine (Linux, MacOS), we have created a simple
+one-liner to get all of those prerequisites installed for you:
+
+```bash
+curl https://getsubstrate.io -sSf | bash -s -- --fast
+```
+
+> **Note:** If you want to see specifically what this script does just visit:
+> https://getsubstrate.io
+
+It will automatically install:
+
+* [CMake](https://cmake.org/install/)
+* [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/)
+* [OpenSSL](https://www.openssl.org/)
+* [Git](https://git-scm.com/downloads)
+* [Rust](https://www.rust-lang.org/tools/install)
+
+If you are using Windows and do not have the [Windows Subsystem for
+Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10), the process
+is a bit harder, but well documented [here](getting-started.md).
+
+### Front-End Development
+
+This tutorial also uses a custom ReactJS front-end which we will modify for
+interacting with our custom Substrate blockchain.
+
+To use the front-end project, you need to install [Yarn](https://yarnpkg.com),
+which may also require you separately install [Node.js](https://nodejs.org/).
+
+* [Install Node.js](https://nodejs.org/en/download/)
+* [Install Yarn](https://yarnpkg.com/lang/en/docs/install/)
+
+## Compiling Substrate
+
+We have created simple template projects to help you get started building on
+Substrate.
+
+1. Clone the Substrate Node Template and Substrate Front-End Template
+
+    ```bash
+    git clone https://github.com/substrate-developer-hub/substrate-node-template
+    git clone https://github.com/substrate-developer-hub/substrate-front-end-template
+    ```
+
+2. Initialize your Wasm build environment
+
+    ```bash
+    # Update Rust
+    rustup update nightly
+    rustup update stable
+
+    # Add Wasm target
+    rustup target add wasm32-unknown-unknown --toolchain nightly
+
+    # Install `wasm-gc` to slim Wasm binaries
+    cargo +nightly install --git https://github.com/alexcrichton/wasm-gc --force
+    ```
+
+3. Compile your Substrate node
+
+    ```bash
+    cd substrate-node-template/
+    cargo build --release
+    ```
```

Comment:
Copied, sure. Exported, nope.

From our contribution guidelines:

> Tutorials should be self-contained, avoid creating links to other tutorials, other how-tos or other information that might have a user leave the actual tutorial. It is OK to provide links to conceptual documents that can enhance a user's knowledge for a given topic.

I think we need to understand our intended audience for these tutorial content, and understand what can be written without having the user "leave" the tutorial.

For example, this tutorial which is a "first substrate chain" where we expect the user to have never used Substrate before, we can assume any link to a setup guide would cause nearly 100% to leave the page.

On the other hand, a tutorial like "building a UTXO chain" would be fair to assume the user already has set up substrate developer environment in the past, and thus we could omit re-writing the instructions in favor of a link, which will likely not have the user leave the page.